React/Modules/RCTUIManager.m

```diff
@@ -78,8 +78,14 @@ static UIViewAnimationOptions UIViewAnimationOptionsFromRCTAnimationType(RCTAnim
     case RCTAnimationTypeEaseInEaseOut:
       return UIViewAnimationOptionCurveEaseInOut;
     case RCTAnimationTypeKeyboard:
-      // http://stackoverflow.com/questions/18870447/how-to-use-the-default-ios7-uianimation-curve
+      // http://stackoverflow.com/questions/18870447/how-to-use-the-default-ios7-uianimation-curve/19439283#19439283
```

Comment:
Can you add a log message (RCTLogWarn) when "keyboard" is specified so that we can deprecate and later remove this value, unless you can think of a reason to keep it around?